Automattic/woocommerce-payments
# Add/stripelink in blocks

Fixes #4236 

#### Changes proposed in this Pull Request


Introduce StripeLink into WooCommerce blocks

Stripe Link payment method implemented in #4106 must work also for blocks checkout .




#### Testing instructions





* Make sure that the classic checkout is not broken. This includes Stripe Link payment method.
* Install https://github.com/woocommerce/woocommerce-blocks plugin
* Create a new checkout page with it
* Retry all of the tests from the classic checkout within the checkout block.

-------------------


**Post merge**
